tests/test_utils.py

```diff
@@ -82,9 +82,20 @@
             ],
         ),
     ),
+    (
+        "CREATE TABLE something (\n  column1, -- comment\n  column2\n)",
+        (
+            None,
+            [
+                ("column1", ""),
+                ("column2", ""),
+            ],
+            [],
+        ),
+    ),
```

Comment:
Could you add a test-case where DB and/or column names have a dash in them?

`"CREATE TABLE foo (\n    "column-1",\n    "column-2"\n)"`